tests/appsec/iast/source/test_body.py

```diff
@@ -28,6 +28,7 @@ def test_source_reported(self):
     )
     @bug(context.library >= "java@1.13.0" and context.library < "java@1.17.0", reason="Not reported")
     @missing_feature(library="dotnet", reason="Not implemented yet")
+    @missing_feature(library="python", reason="Not implemented yet")
```

Comment:
With the changes of this PR, metrics tests expect to fail if the feature test is failing. I added the same `@bug` that the test already have for the test also in telemetry tests.